tests/llmcompressor/transformers/gptq/test_oneshot.py

```diff
@@ -75,7 +75,6 @@ def test_oneshot_application(self):
             model=self.model,
             dataset=self.dataset,
             output_dir=self.output,
-            overwrite_output_dir=True,
```

Comment:
Only used in HF training args
https://github.com/huggingface/transformers/blob/main/src/transformers/training_args.py#L819